src/test/resources/features/AgreeProposal.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Agree Proposal
+  
+  Background:
+    Given There is no registered user with username "user"
+    And I'm not logged in
+    When I register a new user with username "user", email "user@sample.app" and password "password"
+    Then The response code is 201
+    And It has been created a user with username "user" and email "user@sample.app", the password is not returned
+    And I can login with username "user" and password "password"
+
+  Scenario: Agree to an already created agree
+    Given I can login with username "user" and password "password"
+    And There already is a proposal
+    When There already is an agree with the following details:
+      | proposalId  | 1     |
```

Comment:
Like in the other PR, try to avoid using IDs and use entity attributes to retrieve them using a findBy